File: java/Day3.java

import java.util.Scanner;

public class Day3 {
    // for day 3 simple calculator
    public static void main(String[] args) {
        System.out.println("this is day 3");
        
        System.out.println("enter the first number: ");
        // taking input from user

        Scanner a = new Scanner(System.in);
        int num1 = a.nextInt();
        System.out.println("enter the second number: ");
        int num2 = a.nextInt();

        // performing arithmetic operations
        System.out.println("\nthe results are as follows:\n");
        System.out.println("addition:");
        System.out.println(num1 + num2 + "\n");
        System.out.println("subtraction:");
        System.out.println(num1 - num2 + "\n");
        System.out.println("multiplication:");
        System.out.println(num1 * num2 + "\n");
        System.out.println("division:");
        System.out.println(num1 / num2 + "\n");
        a.close();
    }
}

File: c++/day3.cpp
#include <iostream>
using namespace std;

int main() {
    int num1,num2,num3;

    cout << "Enter the first number: " << endl;
    cin >> num1;
    cout << "enter the second number: " << endl;
    cin >> num2;
    cout << "do you want to (1)add , (2)subtract , (3)multiply or (4)divide? " << endl;
    int op;
    cin >> op;

    switch (op)
    {
    case 1:
        num3 = num1 + num2;
        break;
    case 2:
        num3 = num1 - num2;
        break;
    case 3:
        num3 = num1 * num2;
        break;
    case 4:
        if (num2 != 0) {
            num3 = num1 / num2;
        } else {
            cout << "Error: Division by zero is not allowed."<< endl;
        }
        break;
    }
    cout << "The result is: " << num3 << endl;
    return 0;
}
